integration-testing/build.gradle

```diff
@@ -50,12 +50,8 @@ repositories {
 }
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_11
-    targetCompatibility = JavaVersion.VERSION_11
-}
-
-kotlin {
-    jvmToolchain(11)
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
Hmm, still don't quite get why there's a change here. Is something changed in the base branch?

Reply:
Also the build is failing likely because of this change

Reply:
My bad :(( Just reverted the commit and didn't notice, that old jdk 8 targets are still there. Fixed it now.